bundles/org.openhab.ui/web/src/assets/definitions/widgets/system/context.js

```diff
@@ -1,7 +1,7 @@
 import { pt, pb, pi, pn } from '../helpers.js'
 
 export default () => [
-  pt('functions', 'Widget Functions', 'Object with key:arrow-function pairs. Functions are available to expressions in all child components via the <code>fn</code> object.'),
-  pt('constants', 'Widget Constants', 'Object with key:constant pairs. Constants are available to expressions in all child components via the <code>const</code> object.'),
-  pt('variables', 'Widget Variables', 'Object with key:variable default value pairs. Variables are available to expressions in all child components via the <code>vars</code> object and take precedence over variables with the same name from higher contexts.')
+  pt('functions', 'Widget Functions', 'Object with key:arrow-function pairs. Functions are available to expressions in all child components via the <code>fn</code> object.<br />Functions are evaluated on every call and re-evaluated when dependent values change.'),
```

Comment:
```suggestion
  pt('functions', 'Widget Functions', 'Object with key:arrow-function pairs. Functions are available to expressions in all child components via the <code>fn</code> object.<br />Functions are evaluated on every call and re-evaluated when dependency values change.'),
```

I am not native English speaker, but I think this difference is important:

- dependent values: values depending on the function
- dependency values: values the function depends on

The function re-evaluates on the latter.